pkg/config/config.go

```diff
@@ -299,6 +302,8 @@ func InitConfig(config Config) {
 	// Remote config products
 	config.BindEnvAndSetDefault("remote_configuration.apm_sampling.enabled", true)
 	config.BindEnvAndSetDefault("remote_configuration.agent_integrations.enabled", false)
+	config.BindEnvAndSetDefault("remote_configuration.agent_integrations.allow_list", []string{})
+	config.BindEnvAndSetDefault("remote_configuration.agent_integrations.block_list", []string{})
```

Comment:
I don't understand what the purpose of the `block_list` is, if you want to "block" an integration you just shouldn't enable it right ?

Reply:
Block will be to remove any default integration as it might be easier to setup and more future proof